### ISSUE TITLE
Ambilight

### ISSUE THREAD

**Issue by Suyashpoddar** [mpolednik/script.kodi.hue.ambilight#58]
having issue with the Ambilight with bulbs...
It changes them to one color and then dosent adjust to whats on screen...
Running it with Kodi 17 on a shield

**Comment by JesCR** [comment]
turn off "mediacodec(surface)" in teh settings, hardware acceleration section

**Comment by Suyashpoddar** [comment]
Thanks that worked... Not an ideal solution as it can't play 4k hdr videos
then....

On Apr 15, 2017 4:54 AM, "JesCR" <notifications@github.com> wrote:

turn off "mediacodec(surface)" in teh settings, hardware acceleration
section

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/mpolednik/script.kodi.hue.ambilight/issues/58#issuecomment-294280949>,
or mute the thread

**Comment by KillerKelvUK** [comment]
Had this same issue.

Is a fix possible to get ambilight working with hardware acceleration so 4K output works also?

**Comment by mpolednik** [comment]
I don't think so, or at least I'm not sure how to go about it. If we don't get the image, there is nothing to calculate the colors.

**Comment by Suyashpoddar** [comment]
Thoughts on a separate app that can run in the background and read the
screen?

**Comment by KillerKelvUK** [comment]
My setup isn't 4k yet so am wondering if this is a nuance of a 1080 setup and kodi's enabled (by default) but unused hardware acceleration for 4k.  Has anyone tested/verified with a true 4k setup that this doesn't work?

**Comment by andreasbisiach** [comment]
I have a 4K setup (samsung TV and Amazon Fire TV), if I can be of any help in testing I would be glad to. Just need a bit of instructions :-)

**Comment by mpolednik** [comment]
@Suyashpoddar you can always try https://github.com/kershner/screenbloom which should just grab the screen on the background. I haven't ever used it, so ymmv. :)

**Comment by Suyashpoddar** [comment]
I have used screen bloom but I prefer to run my media through my nvidia
shield
